### PR TITLE
Fix empirical distributions and hardcoded gwb prior draws

### PR DESCRIPTION
**Pull request by AaronDJohnson**
Changes:
* GWB prior draws changed to allow for priors that go outside (-18, -14) in `log10_A_gw`.
* Empirical distributions for `ndim != 1` now return proper values for `lqxy` when prior extends beyond empirical distribution support.

### CODE REVIEW

**Comment by AaronDJohnson** [comment]
Fixes #155

File: enterprise_extensions/sampler.py [hunk 1]
```diff
@@ -190,6 +190,11 @@ def draw_from_empirical_distr(self, x, iter, beta):
                 idx = self.pnames.index(self.empirical_distr[distr_idx].param_name)
                 q[idx] = self.empirical_distr[distr_idx].draw()
 
+                # TODO (Aaron): test this:
```
Comment (paulthebaker) [inline enterprise_extensions/sampler.py]: Is this vestigial code that can be deleted?

**Review by paulthebaker — CHANGES_REQUESTED** [review]

File: enterprise_extensions/sampler.py (hunk 1 above)
Comment (AaronDJohnson) [reply enterprise_extensions/sampler.py]: This segment has been tested and the line here has been removed.

File: enterprise_extensions/sampler.py [hunk 2]
```diff
@@ -369,16 +377,21 @@ def draw_from_chrom_gp_prior(self, x, iter, beta):
...
-                   if ('gw' in par and 'log10_A' in par)]
-        gw_par = np.random.choice(gw_pars)
-        idx = self.pnames.index(gw_par)
+        signal_name = [par for par in self.pnames
+                       if ('gw' in par and 'log10_A' in par)][0]
+        idx = list(self.pnames).index(signal_name)
+        param = self.params[idx]
 
-        q[idx] = np.random.uniform(-18, -14)
+        q[self.pmap[str(param)]] = param.sample()
```
[the remark this answers is not in the document]
Comment (AaronDJohnson) [reply enterprise_extensions/sampler.py]: I've gone with the second suggestion in 055b247a7f16ea48639f57c9751808e6392ae96d. Nice catch!

**Comment by paulthebaker** [comment]
I would really like to see some tests for the new functions.  If we don't add tests when new code is added, we're unlikely to ever add them in the future.

**Comment by AaronDJohnson** [comment]
@Hazboun6

**Review by paulthebaker — APPROVED** [review]

**Pull request merged**